config/config.yaml

```diff
@@ -43,7 +43,7 @@ cerberus:
 
 tunings:
     iterations: 5                                        # Iterations to loop before stopping the watch, it will be replaced with infinity when the daemon mode is enabled
-    sleep_time: 60                                       # Sleep duration between each iteration
+    sleep_time: 30                                       # Sleep duration between each iteration
```

Comment:
@paigerube14 hey, wanted to make sure that this change in sleep time is updated by you? Not by the pre-commit.

Other things LGTM!

Reply:
Yes, sorry. That was me but definitely want to keep 60 seconds as the default. Fixed!